src/vue/pages/Fees.vue

```diff
@@ -0,0 +1,168 @@
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="(fee, i) in assetFees" :key="i">
+                <td>
+                  {{ fee.feeType | formatFeeType }}
+                </td>
+                <td>
+                  {{
+                    {
+                      type: fee.feeType,
+                      subtype: fee.subtype }
+                      | formatFeeSubType
+                  }}
+                </td>
+                <td>
+                  {{ {
+                    value: fee.fixed,
+                    currency: fee.feeAsset }
+                    | formatMoney }}
+                </td>
+                <td>
+                  {{ fee.percent | formatPercent }}
+                </td>
+                <td>
+                  {{ fee.lowerBound | formatMoney }}
+                </td>
+                <td>
+                  {{ fee.upperBound | formatMoney }}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div v-else-if="!isFailed">
+    <loader
+      :message-id="'fee-table.lbl-loading'"
+    />
+  </div>
+  <div v-else>
+    <p>
+      {{ 'fee-table.lbl-loading-error' | globalize }}
+    </p>
+  </div>
+</template>
+
+<script>
+import SelectFieldCustom from '../fields/SelectFieldCustom'
+import Loader from '../common/Loader'
+
+import { Sdk } from '@/sdk'
+
+import { mapGetters } from 'vuex'
+import { vuexTypes } from '@/vuex'
+
+export default {
+  name: 'fees',
+  components: {
+    SelectFieldCustom,
+    Loader
+  },
+  data: _ => ({
+    fees: null,
+    isFailed: false,
+    filters: {
+      asset: ''
+    }
+  }),
+  computed: {
+    ...mapGetters('new-wallet', [
+      vuexTypes.wallet
+    ]),
+    assetCodes () {
+      return this.fees !== null
+        ? Object.keys(this.fees).map(asset => asset.toUpperCase())
+        : []
+    },
+    assetFees () {
+      return this.fees !== null
+        ? this.fees[this.filters.asset.toLowerCase()]
+        : []
+    }
+  },
+  async created () {
+    await this.loadFees()
+    if (this.assetCodes.length > 0) {
+      this.filters.asset = this.assetCodes[0]
+    }
+  },
+  methods: {
+    async loadFees () {
+      try {
+        const response = await Sdk.horizon.fees.getAll({
+          account_id: this[vuexTypes.wallet].accountId
+        })
+        this.fees = response.data.fees
+      } catch (error) {
+        this.isFailed = true
+        console.error(error)
+      }
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/scss/variables";
+
+.fee {
+  width: 100%;
+  max-width: 105 * $point;
+}
+
+.fee__asset-info {
+  margin: 2.1 * $point 0;
+}
+
+.fees__table {
+  display: flex;
+  flex-flow: column wrap;
+  overflow-x: auto;
+  box-shadow: 0 0.6 * $point $point 0 $col-table-shadow;
+
+  .fees__table-content {
+    flex: 1;
+    overflow-x: auto;
+    transition: height .3s;
```

Comment:
as I remember, it won't work if the height is not specified. And again, we don't need transitions here at all

Reply:
btw, why can't we merge the `fees__table` and `fees__table-content` tags?

Reply:
Resolved, updated